service/src/main/resources/schema/development/schema.json

```diff
@@ -152,7 +165,7 @@
           "enum": ["bigquery", "firestore", "bucket"]
         },
         "cloudPlatform": {
-          "enum": ["gcp"]
+          "enum": ["gcp", "azure"]
```

Comment:
If we are adding azure, we probably need to add more enums to cloudResource and region? I don't know what those are, but we should check if we are adding the azure as a platform.